app-frontend/src/app/pages/projects/edit/edit.html

```diff
@@ -1,72 +1,110 @@
 <div class="container column-stretch container-not-scrollable">
   <div class="sidebar" ui-view>
-    <div class="sidebar-project">
+
+    <!-- Sidebar static / non scrolling -->
+    <div class="sidebar-static">
+      <!-- Sidebar Header -->
       <div class="sidebar-header">
         <h5 class="sidebar-title">Editing project</h5>
         <div ng-if="$ctrl.project.isAOIProject" class="inline-tag aoi-tag">AOI Project</div>
-        <div ng-if="$ctrl.project.isAOIProject" class="header-controls">
+        <div ng-if="$ctrl.project.isAOIProject" class="sidebar-actions">
           <a ui-sref="projects.edit.aoi-parameters">Edit AOI Parameters</a>
         </div>
       </div>
-      <ul class="sidebar-list">
-        <li ng-if="$ctrl.project.isAOIProject && $ctrl.pendingSceneList.length">
-          <div class="banner banner-secondary">
-            <div>{{$ctrl.pendingSceneList.length}} scenes awaiting approval</div>
-            <div>
-              <button class="btn btn-border-light" ui-sref="projects.edit.aoi-approve">Review Scenes</button>
-            </div>
+      <!-- Sidebar Header -->
+
+      <!-- Alert -->
+      <div ng-if="$ctrl.project.isAOIProject && $ctrl.pendingSceneList.length" class="content">
```

Comment:
(1)
The containers for these alerts have different padding which is making the alert change sizes when transitioning from the main edit page to the add/remove scenes page.